.github/workflows/deploy.yaml

```
@@ -0,0 +1,62 @@
+name: Build and Deploy
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)